### DISCUSSION TITLE
Wolframite v1 talk

### DISCUSSION THREAD

**Discussion by light-matters** [scicloj/wolframite#145] [Ideas]
Goal: Raise awareness of Wolframite, get people inspired to try it, and equip them to know how to go about it.

1) Motivational demo
Start with (the beginning of) this:
https://youtu.be/HFQWNhtPbOo ([slides](https://docs.google.com/presentation/d/1LDv3-cA2BMyDbtKMP6NdR3GtWPNrfLCTCMhfqOlqApM/edit?usp=sharing)) - a short demo of how highlevel and powerful wolfram is

2) A quick intro into Wolframite and how to use it

3) (?) Wolframite for Science - outlook at building efficient data processing pipelines for writing scientific papers

## TODO: Prepare also a workshop

- [ ] Find a topic - an approachable, semi-real problem that can be well solved with Wolfram (which would be much more involved elsewhere)
- [ ] Translate it into Wolframite
- [ ] Prepare the Workshop

**Comment by light-matters** [comment]
# ReClojure 2025 - May 26th, London

Applied here
https://docs.google.com/forms/d/e/1FAIpQLScsGnf4RH4cOlKhLS9pbaXHHA8ym4kHYvUpE4oyKMNNYgQxIw/viewform?pli=1

See https://github.com/scicloj/wolframite/wiki/reClojure-Talk

**Comment by light-matters** [comment]
Draft abstract:
title
Wolframite: Clojure and Wolfram, combined

description
First and foremost, we motivate and review the recent work done to resurrect and extend the Clojure-Mathematica bridge and provide a comprehensive demonstration of how one might use the new library in practice: whether you're an everyday programmer, who just wants to edit a video, or a seasoned scientist, who wants to test quantum physics.

Now that the Wolfram Engine is free for developers however, we take this further and show how by integrating the Wolfram language into a scicloj workflow, we can create an incredibly competitive scientific pipeline. There is a large section of the numerical scientific community who are not programmers but who rely on tools like Mathematica and Matlab and so interop in these areas with a well designed, more comfortable general language would bring that language to a vast new audience. Why shouldn't that language be Clojure?

In short, we want to summarize modern mathematical Clojure and introduce you to Wolframite 1.0, a digital Ferberite that combines the best of a functional general purpose programming language with an unparalleled functional library. Wolframite and scicloj are now ready for adoption and discussion: leaving plenty of room for questions since, as we all know, iron sharpens iron.

**Comment by light-matters** [comment]
Goals:

- Raise awareness
- Motivate people to try it
- Advertise the (good) documentation
- Advertise future workshop
